app/routes/people/person/personal-information.js

```diff
@@ -11,11 +17,99 @@ export default class PeoplePersonPersonalInformationRoute extends Route {
       include: [
         'mandatories.mandate.governing-body.is-time-specialization-of.administrative-unit.classification',
         'mandatories.contacts',
+        'agents-in-position',
       ].join(),
     });
+    const contacts = [];
+    const m = await person.mandatories;
+    console.log(m);
+    const mandatories = person.mandatories.filter(
```

Comment:
You can use [`toArray`](https://api.emberjs.com/ember/4.1/classes/EmberArray/methods/toArray?anchor=toArray) to convert an Ember array to a native array.